src/main/java/org/folio/rest/impl/AcquisitionsUnitsAPI.java

```diff
@@ -0,0 +1,63 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.jaxrs.model.AcquisitionsUnit;
+import org.folio.rest.jaxrs.model.AcquisitionsUnitCollection;
+import org.folio.rest.jaxrs.resource.AcquisitionsUnitsStorageUnits;
+import org.folio.rest.persist.EntitiesMetadataHolder;
+import org.folio.rest.persist.PgUtil;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.persist.QueryHolder;
+
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+import static org.folio.rest.persist.HelperUtils.getEntitiesCollection;
+
+public class AcquisitionsUnitsAPI implements AcquisitionsUnitsStorageUnits {
+
+  private static final String ACQUISITIONS_UNITS_TABLE = "acquisitions_units";
+  private String idFieldName = "id";
+
+  public AcquisitionsUnitsAPI(Vertx vertx, String tenantId) {
```

Comment:
As @pautau-s mentioned RMB 25 deprecates some of the things so please remove this constructor.